packages/agoric-cli/template/.agservers/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "agoric-servers",
+  "name": "@agoric/dapp-agoric-servers",
```

Comment:
No, because these packages are both marked as `"private"`, and also because external (non Agoric org) dapp developers don't have permission to publish to `@agoric/`, so it will fail noisily if they remove the `"private"` and try without renaming the package.